recipes/flang-activation/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "19.1.0.rc3" %}
+
+package:
+  name: flang-activation
+  version: {{ version }}
+
+build:
+  number: 0
+  # intentionally only windows (main target) & linux (debuggability)
+  skip: true  # [osx]
+
+outputs:
+  - name: flang_impl_{{ cross_target_platform }}
+    script: install_flang_impl.sh   # [unix]
+    # we don't cross-compile on windows -> no symlinks
+    requirements:
+      - flang ={{ version }}
+      - lld ={{ version }}          # [win]
+      - compiler-rt ={{ version }}
+      - compiler-rt_{{ cross_target_platform }} ={{ version }}
+      # during rc-phase; avoid different rc-builds conflicting
+      - libllvm19 ={{ version }}
+    test:
+      commands:
+        - {{ CBUILD }}-flang --version  # [unix]
+        - {{ CHOST }}-flang --version   # [unix]
+
+  - name: flang_{{ cross_target_platform }}
+    script: install_flang.sh   # [unix]
+    script: install_flang.bat  # [win]
+    run_exports:
+      strong:
+        - libflang >={{ version }}
+    requirements:
+      run:
+        - {{ pin_subpackage("flang_impl_" ~ cross_target_platform, exact=True) }}
+    test:
+      files:
+        - hello_world.f90
+      commands:
+        # rename flang-new -> flang is happening soon:
+        # https://discourse.llvm.org/t/proposal-rename-flang-new-to-flang/69462
```

Comment:
This comment seems irrelevant

Reply:
This is an updated version from what we currently [have](https://github.com/conda-forge/flang-feedstock/blob/35b55990fe7350ff4dcde7a3a1298714ca6f5660/recipe/meta.yaml#L121-L122) in the flang feedstock. As it was looking like it would finally happen a few days ago (until the goalposts shifted yet again), I was debating with myself whether we want to do the rename for flang 19 already. Might make the fortran migration a bit simpler. 🤷

Reply:
I didn't mean the contents of the comment. I was referring to why this comment needs to be in the test section. It seems irrelevant.

Reply:
It's more visible in the recipe than in the activation script, and the test section is where we're using the name (or now, the value of `FC`).